flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -519,6 +519,12 @@ object JobManager {
           configuration.setString(ConfigConstants.FLINK_BASE_DIR_PATH_KEY, config.configDir + "/..")
         }
 
+        if (GlobalConfiguration.getString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY,
+          null) == null) {
+          configuration.setString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY,
+            InetAddress.getLocalHost.getHostName)
+        }
+
         val hostname = configuration.getString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY, null)
```

Comment:
Can't we just set InetAddress.getLocalHost.getHostName as the default value here instead of having the if block above?